release.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+RELEASE_DIR='release';
```

Comment:
Let's not make RELEASE_DIR a variable, I'd rather just hard-code the string into the paths